### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by DanCesca**
Hey! With this pull request I have: 

- created the Product Page
- added Select button to Detail Card with link to coffee product page showing more details about the product selected from the detail card
- adjusted the routes accordingly

Thank you very much for your feedback and review!

### CODE REVIEW

File: client/src/pages/product.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import Header from '../components/Header';
+import NavBar from '../components/NavBar';
+import styled from 'styled-components';
+import { useParams } from 'react-router-dom';
+
+function Product({ tastes }) {
+  const { id } = useParams();
+  const taste = tastes.find(taste => taste.id === id);
+
```
Comment (mpagels) [inline client/src/pages/product.jsx:9]: What happens if the `id` is not found? I highly recommend adding a check that returns a visible info for the user, that the `id` was not found.

**Review by mpagels — CHANGES_REQUESTED** [review]
Hey Daniele, 
awesome pull request. I just found an issue that could lead to an error in the app. Pls review my app and let discuss this. I marked the pr as `request changes` so you have the chance to do something about it. It would make the app better ;-)

File: client/src/pages/product.jsx (hunk 1 above)
Comment (DanCesca) [reply client/src/pages/product.jsx:9]: Hey Martin! Thank you again for your valuable feedback! The error is now managed, please confirm so I can merge. Thanks!

**Review by mpagels — APPROVED** [review]
lgtm

**Pull request merged**